cloudwatch_alarm_notifications.tf

```diff
@@ -0,0 +1,27 @@
+module "cloudwatch_alarms_notifications_sns" {
+  source = "git::https://github.com/nationalarchives/da-terraform-modules//sns"
+  sns_policy = templatefile("${path.module}/templates/sns/cloudwatch_alarm_policy.json.tpl", {
+    topic_name           = "${local.environment}-dlq-notifications"
+    account_id           = data.aws_caller_identity.current.account_id
+    cloudwatch_alarm_arn = module.download_files_sqs.dlq_cloudwatch_alarm_arn
+  })
+  tags = {
+    Name = "Preservica Config SNS"
+  }
+  topic_name  = "${local.environment}-dlq-notifications"
+  kms_key_arn = module.dr2_kms_key.kms_key_arn
+  sqs_subscriptions = {
+    dlq_notifications_queue = module.cloudwatch_alarms_notifications_queue.sqs_arn
+  }
+}
+
+module "cloudwatch_alarms_notifications_queue" {
+  source     = "git::https://github.com/nationalarchives/da-terraform-modules//sqs"
+  queue_name = "${local.environment}-dlq-notifications"
```

Comment:
`dlq`?

Reply:
I've renamed this and the topic so it doesn't say dlq any more.